packages/xrpl/test/wallet/index.ts

```diff
@@ -949,6 +949,16 @@ describe('Wallet', function () {
 
       assert.equal(isVerified, false)
     })
+
+    it('returns true when verifying a deserialized Transaction object', function () {
+      const wallet = new Wallet(publicKey, privateKey)
+      const decodedTransaction = decode(
+        prepared.signedTransaction,
+      ) as unknown as Transaction
+      const isVerified: boolean = wallet.verifyTransaction(decodedTransaction)
+
+      assert.equal(isVerified, true)
+    })
```

Comment:
Should there also be a test that fails for an invalid transaction? (Ex. Improper use of string in a number field?)

Reply:
I think that's a broader scope than this PR - we don't do _any_ checks on the transaction data right now.